### PR TITLE
Visualize inputs at a glance

### LINKED ISSUE #64: The definition of certain input and output parameters is not clear
minimal peak power capacity = installed power

Available production technologies = Available generation and storage technologies
Tool has a list of technologies on the market, user inputs list of what is available. Tool would cross some technologies off the list based on requirements.

required size of electrolysers and batteries = required size of storages
Required size of electrolysers, batteries, thermal storages, etc.

### PR DESCRIPTION
**Pull request by TheOneAndra**
**Changes proposed in this pull request**:
- Propose a way to access all inputs quickly from the RDT menu
- Propose a way to organize inputs
- Could be a way to summarize outputs as well

The following steps were realized, as well (if applies):
- [ ] Update the CHANGELOG.md

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

File: docs/input_output/inputs.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            Future grid structure	,	Technical	,		,
+            Security of supply	,	Technical	,		,
+            Balancing ratio for anciliary services	,	Technical	,		,
+            Number of future extra users	,	Technical	,		,
+            Load types	,	Technical	,		,
+            Project lifetime	,	Technical	,		,
+
+
+**Could also be  specified for each if it is a project input or a scenario input**
+**Could also be a chance to explain the default value for each input**
```
Comment (mahendrark) [inline docs/input_output/inputs.rst]: Can you check the rendering of these sentences again?

**Review by mahendrark — APPROVED** [review]

**Comment by mahendrark** [comment]
@TheOneAndra 

Can be merged after you update the CHANGELOG

**Comment by Bachibouzouk** [comment]
Nice idea, but we can do it directly from a file, this way RTD gets updated automatically when we update the file. I will work on that I think it will be more efficient timewise

**Review by Bachibouzouk — CHANGES_REQUESTED** [review]
We need to match the columns there with the attributes of [input_parameter_field](https://github.com/rl-institut/open_plan/blob/dev/docs/view_components/input_parameter_field.rst). BTW this is a good example of "write an issue first, then we'll discuss how to solve it" to provide opportunity to others to tell you if they already have solutions for the issue ;)

File: docs/_files/input_parameters_list.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+,:Default:,:Definition:,:Example:,:Restrictions:,:Type:,:Unit:,label,ref,:Category:
```
Comment (Bachibouzouk) [inline docs/_files/input_parameters_list.csv]: we only need to edit one source file for the inputs

Comment (TheOneAndra) [reply docs/_files/input_parameters_list.csv:1]: How do I unclude the csv under _files in the tool_interface.rst?

File: docs/parameters.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Parameters
+**********
+
+Inputs
+======
+
+Inputs are necessary values for the optimization. All inputs have default values that can be replaced by the user if desired.
+
+
+.. This file is generated automatically when conf.py is executed
```
Comment (TheOneAndra) [inline docs/parameters.rst:13]: I get a synthax error in conf.py (line 69)
any idea why @Bachibouzouk ?

File: docs/parameters.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+   :header-rows: 1
+
+Another way to display the Inputs
+
+.. This file is generated automatically when conf.py is executed
+
+.. include:: input_parameters_list.inc
+
+**Could also be  specified for each if it is a project input or a scenario input**
+**Could also be a chance to explain the default value for each input**
```
Comment (TheOneAndra) [inline docs/parameters.rst]:
```suggestion
```

**Comment by TheOneAndra** [comment]
Change EUR instead of €

**Comment by Bachibouzouk** [comment]
@TheOneAndra - can you make a csv with the parameters from the nextcloud file and commit it? Maybe one for inputs and one for outputs.

**Review by Bachibouzouk — APPROVED** [review]

**Pull request merged**